TWLight/applications/helpers.py

```diff
@@ -170,27 +170,27 @@ def get_output_for_application(app):
 
 
 def get_active_authorizations(partner_pk, stream_pk=None):
```

Comment:
I would have this return the queryset instead of a count. You could add a separate helper for `count_active_authorizations`

Reply:
this was basically missed in the initial review of this code: when you have a method or function named get_xxx_objects it should return a list or queryset of those objects